gse-util/src/main/java/com/powsybl/gse/util/GroovyCodeEditor.java

```diff
@@ -102,6 +122,94 @@ public GroovyCodeEditor(Scene scene) {
         codeArea.setOnDragOver(this::onDragOver);
         codeArea.setOnDragDropped(this::onDragDropped);
         codeArea.setOnSelectionDrag(p -> allowedDrag = true);
+
+        codeArea.caretListener = (observable, oldvalue, newvalue) -> {
+            bracketsMatches.searchingForMatches = false;
+            codeArea.setStyleSpans(0, computeHighlighting(codeArea.getText()));
```

Comment:
Each time the caret change, you highlight and index everything? Do I understand well?